### ISSUE TITLE
Add formatting support for prettier-standard

### ISSUE THREAD

**Issue by bluz71** [jose-elias-alvarez/null-ls.nvim#468]
### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues) and there are no existing ones with the same request.

### Feature description

Hello,

It would be nice to add support for prettier-standard in null-ls.

[prettier-standard](https://github.com/sheerun/prettier-standard) formats JavaScript code via prettierx to [standardjs](https://standardjs.com) rules.

This is the implemention I am using in my own setup:

```lua

M.formatting_prettier_standard = h.make_builtin({
  method = FORMATTING,
  filetypes = {'javascript'},
  generator_opts = {
    command = 'prettier-standard',
    args = {'--stdin'},
    to_stdin = true,
  },
  factory = h.formatter_factory,
})
```

### Help

No

**Comment by jose-elias-alvarez** [comment]
Just saw this issue too. Again, feel free to submit a PR!

**Comment by bluz71** [comment]
Ok.

**Comment by jose-elias-alvarez** [comment]
Closed by #476.
